interlok-elastic-common/src/main/java/com/adaptris/core/elastic/CSVDocumentBuilderImpl.java

```diff
@@ -57,13 +65,28 @@
    * Defaults to {@link BasicFormatBuilder} by default.
    * </p>
    */
-  @NotNull
   @AutoPopulated
   @Valid
   @Getter
   @Setter
-  @NonNull
+  @Deprecated
+  @ConfigDeprecated(/*removalVersion = "x.y.z",*/ message = "Use 'preference' instead", groups = Deprecated.class)
```

Comment:
Is there a version that we'll target for this to be removed?

Reply:
5.0 I think.